internal/github/github_test.go

```diff
@@ -103,9 +105,9 @@ func TestNewGithubClientWithAppAuthentication(t *testing.T) {
 	require.IsTypef(t, &httpcache.Transport{}, nestedTransport.Client.HTTPClient.Transport, "Returned client transport is not a caching transport (is %T)", nestedTransport.Client.HTTPClient.Transport)
 }
 
-// newClientFromMock returns a new GHClient whose transport is configured to use
+// newClientFromMock returns a new REST & GraphQL GHClient whose transport is configured to use
```

Comment:
ok we're doing micro nits now 😈 

```suggestion
// newClientFromMock returns a new REST & GraphQL GHClient whose transports are configured to use
```